tracker/view/login.py

```diff
@@ -44,3 +56,76 @@ def logout():
     user_invalidate(current_user)
     logout_user()
     return redirect(url_for('tracker.index'))
+
+
+@tracker.route('/sso-auth')
```

Comment:
is there an advantage of using another route? What if sso is actually disabled in the config?
Can we condense this into the `login` route somehow? And just do different things based on if SSO is enabled?

Reply:
I created that route because it is used only as the redirection target after the user went to the IDP and authenticated himself.
That means that there will always be access tokens to work with when this route is hit.
We could do two things:
1. Turn this route into a dummy when SSO is off
2. Equip the login route with a parameter that indicates whenever something was redirected from the IDP to the tracker (I think that's the preferrable solution)

Reply:
The second option sounds like a good approach, lets unify the login endpoint